change/@internal-react-composites-4cba40c4-0e66-45b4-9ec3-c580777b3188.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
`prerelease` if it's changing/exposing props from chat-composite.
`none` if all of it is internal logic